hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -164,25 +185,27 @@
         {
           bits: "3:0",
           name: "SEL",
+          mubi: true,
           desc: '''
-            A value of b1010 selects external clock as clock for the system.
+            A value of kMultiBit4True selects external clock as clock for the system.
             While this register can always be programmed, it only takes effect when the system is in
             life cycle TEST or RMA states when DFT is enabled.
 
             All other values are invalid and keep clocks on internal sources.
           '''
-          resval: "0x5"
+          resval: "false"
         },
         {
           bits: "7:4",
-          name: "STEP_DOWN",
+          name: "LOW_SPEED_SEL",
+          mubi: true,
           desc: '''
-            A value of b1010 steps down the clock dividers by a factor of 2 if the !!EXTCLK_CTRL.SEL
-            field is also set to b1010.
+            A value of kMultiBit4True selects low speed external clocks.
+            All other values selects nominal speed clocks.
 
-            All other values have no effect.
+            Note
```

Comment:
Note what? :)

Reply:
uhh....let me go take a look hehe.

Reply:
please have a look at the updated statement, i've also filed #9012.